File: SurgeonPortal.Models/GraduateMedicalEducation/RotationReadOnlyModel.cs
using System;

namespace SurgeonPortal.Models.GraduateMedicalEducation
{
    public class RotationReadOnlyModel
    {
        public int Id { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public string ProgramName { get; set; }
        public string AlternateInstitutionName { get; set; }
        public int ClinicalLevelId { get; set; }
        public string ClinicalLevel { get; set; }
        public bool IsEssential { get; set; }
        public bool IsCredit { get; set; }
        public string Other { get; set; }
        public string NonSurgicalActivity { get; set; }
        public bool IsInternationalRotation { get; set; }
        public string ClinicalActivity { get; set; }
    }
}

File: SurgeonPortal.Api/ClientApp/src/app/api/models/graduatemedicaleducation/rotation-read-only.model.ts

export interface IRotationReadOnlyModel {
    id: number;
    startDate: string;
    endDate: string;
    programName: string;
    alternateInstitutionName: string;
    clinicalLevelId: number;
    clinicalLevel: string;
    isEssential: boolean;
    isCredit: boolean;
    other: string;
    nonSurgicalActivity: string;
    isInternationalRotation: boolean;
    clinicalActivity: string;
}
